### ISSUE TITLE
youtube-search: ctrl+i breaks searching.

### ISSUE THREAD

**Issue by MintMain21** [CogentRedTester/mpv-scripts#12]
Another MPV Plugin (Youtube+Download) uses the crtl+i command (by default) for downloading videos with subtitles embedded. While testing this command for conflicts, I found it summoned the search bar for youtube-search, which is weird because the default command for that (and likewise how I have it set) is ctrl+y. I tried ctrl+y, and it wouldn't summon the search bar. Tried ctrl+i again, and nothing happened. Tried shift+y twice, nothing happened. Closed MPV and restarted it, and shift+y and ctrl+y worked as intended, but using ctrl+i again would summon the search bar randomly, and break the plugin as previously detailed until I refreshed MPV. Likewise, if I use ctr+i before using any other youtube-search commands, no Search Bar will appear, but the ctrl+y command will be broken anyway. 

tldr: using the ctrl+i keyboard command breaks the plugin and makes it impossible to use the search functionality without closing the session and starting a new one.

Looking at the plugin's code, nowhere is Ctrl+i ever mentioned, which makes this even weirder.

**Comment by CogentRedTester** [comment]
I believe it is caused by mpv-user-input. I have updated user-input.lua this [this commit](https://github.com/CogentRedTester/mpv-user-input/commit/7ea20a6b089498d9a250b13ee97ca6e1c10c0dbc), so download the latest version of the file and see if it is now working.

**Comment by MintMain21** [comment]
That fixed it.
